reactive/hibernate-reactive-2/src/test/java/org/hibernate/bugs/ConnectionURL.java

```diff
@@ -0,0 +1,16 @@
+package org.hibernate.bugs;
+
+/**
+ * JDBC URL connection strings for the property {@link org.hibernate.reactive.provider.Settings#URL} when the database
+ * is started following the instructions on the
+ * <a href="https://github.com/hibernate/hibernate-reactive/blob/main/podman.md#how-to-start-the-test-databases-using-podman">Hibernate Reactive GitHub repository</a>.
```

Comment:
Hey @DavideD ,

IDK if it makes sense for Reactive or not, but just in case 😃... for Search, we've been using test containers to start DB/ES instances. And we also did it here for the test temaple: https://github.com/hibernate/hibernate-test-case-templates/blob/e833f43bcfb0fffc7ab9dc300e7cd7a3183d05f5/search/hibernate-search-7/orm-elasticsearch/src/test/java/org/hibernate/search/bugs/SearchBackendContainer.java#L10

Reply:
Yes, we use it in Hibernate Reactive too.
I was trying to avoid extra complexity in the template and let people start the database they way they prefer.

But maybe I'm overthinking it. 
I will see what it looks like adding it here.

Reply:
> But maybe I'm overthinking it.

or maybe I am 😃